common-util/src/main/java/org/sunbird/services/sso/impl/KeyCloakRsaKeyFetcher.java

```diff
@@ -23,16 +23,23 @@
 import org.sunbird.common.models.util.ProjectLogger;
 import org.sunbird.common.models.util.PropertiesCache;
 
-/** Class to fetch SSO public key from Keycloak server using 'certs' HTTP API call. */
+/**
+ * author : github/iostream04 Class to fetch SSO public key from Keycloak server using 'certs' HTTP
+ * API call.
+ */
 public class KeyCloakRsaKeyFetcher {
 
+  /*
+  * This method will connect to server get modulus, exponent values and genrated publickey
```

Comment:
Update with below comment.
/**
*This method will accept keycloak baseUrl and realm name, Based on provided values it will fetch PublicKey from keycloak.
* @param url,  A string value having keycloak baseUrl
* @param realm , keycloak realm name
 *  @return PublicKey  , used to verify user access token.
*/